### ISSUE TITLE
Both 4.2.0 versions will stop if they encounter a deprecated epub:type, won’t check aria role is correct

### ISSUE THREAD

**Issue by JayPanoz** [w3c/epubcheck#1021]
So sorry to be that guy. :-(

In an EPUB I’ve been working on for a few weeks, I temporarily used a placeholder like: 

```
<div epub:type="rearnote" role="doc-endnote">
```

Then I reported issue #1010 so I wanted to come back to it and correct that because `doc-endnote` require `<li>` and, out of curiosity, I tested version 4.2.0

In epubcheck 4.2.0 beta I got this warning: 

```
Property 'rearnote' is deprecated. Consider using the 'endnote' property instead.
```

In epubcheck 4.2.0 rc, it went completely unnoticed – since handling of deprecated epub:types was changed somehow?

So as far as I can tell, epubcheck may be stopping validation on the deprecated epub:types and not check the `role` that follows. 

Logs: 

- [4.2.0-beta.log](https://github.com/w3c/epubcheck/files/2999612/4.2.0-beta.log)
- [4.2.0-rc.log](https://github.com/w3c/epubcheck/files/2999613/4.2.0-rc.log)

**Comment by rdeltour** [comment]
So there are two different things here:

- all messages related to `epub:type` values are now reported as `USAGE` (since #1013, following a decision made in the [EPUB 3 CG call of March 14, 2019](https://www.w3.org/2019/03/14-epub3cg-minutes.html)). In other words, the message you got in in v4.2.0-beta is still reported (with the same wording), but at the `USAGE` level.
- the `doc-endnote` role is not reported as it is [allowed on `div` elements](https://www.w3.org/TR/html-aria/#div) (I guess the confusion comes from the role being explicitly listed as allowed on `li` elements… but note that it is also implicitly allowed wherever "Any `role`" is allowed, which is the case of `div` elements).

So I think there are nothing wrong with EPUBCheck here (no false-negative). I'm closing this as `invalid`, but feel free to keep discussing or reopen if I missed something!

**Comment by JayPanoz** [comment]
Eew thanks, sorry for the inconvenience.

God I'm happy this is my last eprdctn gig, I can't even tell whether the most basic a11y stuff I'm doing is right or not anymore. 😩

**Comment by rdeltour** [comment]
> Eew thanks, sorry for the inconvenience

No problem! ☺️
The ´epub:type’/ARIA role mapping is a bit of a red herring (in that it’s not really a mapping), so it’s prone to confusion…
